### PR TITLE
Fix the conversations icons

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: src/components/Navigation/ConversationsList/ConversationsIcon.vue [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<!--
+  - @copyright Copyright (c) 2019 Marco Ambrosini <marcoambrosini@pm.me>
```
Comment (ma12-co) [inline src/components/Navigation/ConversationsList/ConversationsIcon.vue]: Joas Shilling

File: src/components/Navigation/ConversationsList/ConversationsIcon.vue [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	<div v-if="item.type === 4"
+		class="avatar icon icon-changelog" />
+	<!--	<div v-if="item.type === 2"-->
+	<!--		class="avatar icon icon-contacts" />-->
+	<!--	<div v-if="item.type === 3"-->
+	<!--		class="avatar icon icon-public" />-->
+	<Avatar v-else
+		:size="44"
+		:user="item.displayName"
+		:display-name="item.displayName" />
```
Comment (ma12-co) [inline src/components/Navigation/ConversationsList/ConversationsIcon.vue]: Here we also need to pass i the image url, if there's one

File: src/components/Navigation/ConversationsList/ConversationsIcon.vue [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+	<div v-if="item.objectType === 'file'"
+		class="avatar icon icon-file" />
+	<div v-else-if="item.objectType === 'share:password'"
+		class="avatar icon icon-password" />
+	<div v-else-if="item.type === 4"
+		class="avatar icon icon-changelog" />
+	<div v-else-if="item.type === 2"
+		class="avatar icon icon-contacts" />
+	<div v-else-if="item.type === 3"
+		class="avatar icon icon-public" />
```
Comment (ma12-co) [inline src/components/Navigation/ConversationsList/ConversationsIcon.vue]: maybe it's cleaner if we do all these checks in a computed property and use only one div element:

```
<div class="avatar icon" v-bind:class="iconType" />
```
And in computed:
```
computed: {
    iconType() {
        if (item.objectType === 'file) {
            return 'icon-file'
            }  //and so on
    }
}
```

Comment (nickvergessen) [reply src/components/Navigation/ConversationsList/ConversationsIcon.vue]: done

**Comment by ma12-co** [comment]
@nickvergessen could oyu create a `ConversationsIcon` folder in `AppContentListItem` and put your `ConversationsIcon.vue` there?

**Comment by nickvergessen** [comment]
Renamed to ConversationIcon as discussed in #2311

**Pull request merged**